136-git-sync-manager/src/main/java/io/harness/gitsync/gitxwebhooks/service/GitXWebhookServiceImpl.java

```diff
@@ -130,14 +132,16 @@ public Optional<GetGitXWebhookResponseDTO> getGitXWebhook(GetGitXWebhookRequestD
         }
         List<GetGitXWebhookResponseDTO> getGitXWebhookResponseList = prepareGitXWebhooks(gitXWebhookList);
         if (getGitXWebhookResponseList.size() > 1) {
+          log.error(String.format(
+              "For the given key with accountIdentifier %s and gitXWebhookIdentifier %s found more than one unique record.",
```

Comment:
```suggestion
              "Found more than one unique record for the given key with accountIdentifier %s and gitXWebhookIdentifier %s.",
```
(Do we not want to write what component found more than one unique record? If so, incorporate this suggestion. Else, retain current structure and name the actor.)